test/test_utils.cpp

```diff
@@ -0,0 +1,17 @@
+#include <gtest/gtest.h>
+
+#include "utils.hpp"
+
+TEST(getFileExtension, NoExtension)
+{
+	EXPECT_EQ(utils::getFileExtension("file"), "");
+	EXPECT_EQ(utils::getFileExtension("some/path/to/file"), "");
+	EXPECT_EQ(utils::getFileExtension("a/path/with.inbetween/to/file"), "");
```

Comment:
Ok, I see now why you check against the dir in the function. The previous this regarding comment can be ignored.

Reply:
Why not just delete the comment :o

Reply:
To have more of an exchange of thoughts!